core/src/main/java/org/apache/carbondata/core/util/path/CarbonTablePath.java

```diff
@@ -303,6 +303,20 @@ public String getCarbonIndexFilePath(String taskId, String partitionId, String s
     }
   }
 
+  public String getCarbonIndexFilePath(String taskId, String partitionId, String segmentId,
+      int batchNo, String bucketNumber, String timeStamp,
+      ColumnarFormatVersion columnarFormatVersion) {
+    switch (columnarFormatVersion) {
+      case V1:
+      case V2:
+        return getCarbonIndexFilePath(taskId, partitionId, segmentId, bucketNumber);
+      default:
```

Comment:
Whats the different between V1/V2 and default? 

Does V3 acts the same as default?